spacepy/pybats/__init__.py

```diff
@@ -323,19 +220,19 @@ def add_planet(ax, rad=1.0, ang=0.0, add_night=True, zorder=1000,
 
     Parameters
     ==========
-    ax : Matplotlib Axes object
+    ax - Matplotlib Axes object
        Set the axes on which to place planet.
 
     Other Parameters
     ================
-    rad : float
+    rad - float
        Set radius of planet.  Defaults to 1.
-    ang : float
+    ang - float
        Set the rotation of the day-night terminator from the y-axis, in degrees
        Defaults to zero (terminator is aligned with Y-axis.)
-    add_night : boolean
+    add_night - boolean
        Add night hemisphere.  Defaults to **True**
-    zorder : int
+    zorder - int
        Set the matplotlib zorder of the patch to set how other plot
        elements order with the inner boundary patch. Defaults to 1000,
        nightside patch is given zorder of *zorder+5*.
```

Comment:
I'm not going to point all these out, please go through the diffs and find them. Then rebase into the correct spot so that they never happened in history. Or you can rebase --interactive and label the commits where these were introduced as "edit". When the rebase reaches that commit, edit this out, do `commit --amend`, and then continue the rebase.

Reply:
resolved